l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -721,8 +722,12 @@ def setDatiPagamento(self, invoice, body):
     def setAttachments(self, invoice, body):
         if invoice.fatturapa_doc_attachments:
             for doc_id in invoice.fatturapa_doc_attachments:
+                nome_attachment = doc_id.datas_fname if len(
+                    doc_id.datas_fname) <= 60 else os.path.join([
+                        os.path.splitext(doc_id.name)[0][:55],
```

Comment:
Assuming there will not be extensions longer than 5 chars?

Reply:
is it possible? I change the function